packages/@aws-cdk/custom-resources/lib/aws-custom-resource/aws-custom-resource.ts

```diff
@@ -128,9 +128,17 @@ export interface AwsCustomResourceProps {
    */
   readonly onDelete?: AwsSdkCall;
 
+  /**
+   * Allow using '*' (star) as the resources for the SDK calls auto-generated policy statements.
+   *
+   * @default false
+   */
+  readonly allowStarPermissions?: boolean;
```

Comment:
Is “star permissions” a commonly used term?

Also, This is a semantic opportunity to indicate permissions are generated from actions so I would name this “generateStarPermissions”. 

 A user that is exposed to this api for the first time doesn’t know that permissions can be generated from the specified actions.

Reply:
Outdated - resolving